### PR TITLE
English improvements

### PR DESCRIPTION
**Pull request by michaelarnauts**
### Check if this PR fulfills these requirements:
* [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [X] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [X] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [X] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [X] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

While going over the Dutch translation, I've noticed some odd English strings.
* Use `'My list'` consistently.
* Don't use `shared-library`
* Used `…`, `»` and `«` UTF-8 characters.

I'm actually thinking about the lowercase issue, English isn't my native language, but from what I can find online, we don't need to capitalise so much words (like you described here: https://github.com/CastagnaIT/plugin.video.netflix/pull/830#issuecomment-686683046). I could use some input from native English speakers here.

Feel free to comment on what you like and what you don't.

### CODE REVIEW

**Comment by michaelarnauts** [comment]
I'm aware that this will require changes to all other language files, since Kodi doesn't match by ID unfortunately.

**Comment by CastagnaIT** [comment]
I don't know English well either...
until now to me this look well
but before i will merge the romanian pr

**Comment by dagwieers** [comment]
I would replace ampersand with "and".

File: resources/language/resource.language.en_gb/strings.po
Comment (tweimer) [inline resources/language/resource.language.en_gb/strings.po:152]: Eh, I do not like that special char. I would stick to "...".

**Review by tweimer — CHANGES_REQUESTED** [review]
I disagree with the UTF-8 chars. I prefer the current form. The other changes are fine for me.

**Comment by tweimer** [comment]
@dagwieers You mean strings "(between 0 & 10)" and "Show trailers & more"? I agree, "and would look better...

**Comment by dagwieers** [comment]
> I disagree with the UTF-8 chars. I prefer the current form. The other changes are fine for me.

I prefer the UTF-8 characters, but do not think it is worth discussing :-) Rather have @CastagnaIT decide.

**Comment by dagwieers** [comment]
PS This PR should also includes the updates for the existing languages. I understand we would not do that straight away until we have settled what we want to change, but we shouldn't merge until the english strings have been updated everywhere.

**Comment by michaelarnauts** [comment]
I don't really mind about the UTF-8 characters, except that they look a bit nicer IMO. But I would also say it's up to @CastagnaIT.

**Comment by michaelarnauts** [comment]
@dagwieers I was hoping there would be a command line tool to update existing language files by the id or something.

**Comment by CastagnaIT** [comment]
> I disagree with the UTF-8 chars.

mmh yes becouse look better and no for other reasons

I would also prefer normal characters, because most of the time you do not use non-standard characters in sentences
for example Asian languages can use different punctuation and this form may not be respected

for the dots "…" perhaps is better use the normal dot char "..."

for the »« in my opinion could be maintained as UTF8, is only an "arrow"

> PS This PR should also includes the updates for the existing languages. 
@dagwieers yes but it is better before define what changes to do

**Comment by michaelarnauts** [comment]
Is there something we should do about the Capitalisation of nouns? :)

**Comment by michaelarnauts** [comment]
Okay, I've updated the PR with the requested changes. 

Regarding the capitalisation, it seems that there are different rules for titles as for normal text.
-> https://www.grammarly.com/blog/capitalization-rules (see point 6).

**Comment by CastagnaIT** [comment]
I leave it to you to speak, with pure English i would only make things worse
i help myself with the translator :D

a little thing perhaps:
tv shows > TV show

**Comment by CastagnaIT** [comment]
next days i will publish a minor release
you can end here and complete the changes in relative translations,
or if you want you can wait other days if there are other changes to do

**Comment by michaelarnauts** [comment]
I have made a small script that use polib to update the other language files when translations are added/modified in the english file. It's similar to msgmerge, but it does the matching on `msgctxt`. The only drawback is that it puts strings with newlines on a new line, but this is also done for the header, so maybe this doesn't matter.

@CastagnaIT how do you do this when you add a translation? Do you just change all the files and add it there manually, or do you also have a script to do this?

**Comment by CastagnaIT** [comment]
> I have made a small script that use polib to update the other language files when translations are added/modified in the english file. It's similar to msgmerge, but it does the matching on `msgctxt`. The only drawback is that it puts strings with newlines on a new line, but this is also done for the header, so maybe this doesn't matter.
> 
> @CastagnaIT how do you do this when you add a translation? Do you just change all the files and add it there manually, or do you also have a script to do this?

usually i use a software to compare the files and merge the change manually
i do not like these newlines make the files difficult to compare manually and in some case also to read

but i think can be resolved easly
so i was waiting for your modifications to make another one
but at this point you can make this change for me and then solve everything

my change is replace all return chars escape with Kodi labeling:
`\r\n` with `[CR]`
`\n` with `[CR]` (some id have only `\n`)

in this way the script should not create new lines
this change is necessary because some languages "right to left", can have strange behaviors with the return char escapes

**Comment by michaelarnauts** [comment]
@CastagnaIT I've replaced all newlines with `[CR]` (except in the header). I've also replaced the << and >> characters in the other languages.

**Comment by CastagnaIT** [comment]
nicework! now it is very clear

File: resources/language/resource.language.hu_hu/strings.po
Comment (CastagnaIT) [inline resources/language/resource.language.hu_hu/strings.po:12]: only a minor thing here the original header metadata is broken and some are missing

Comment (michaelarnauts) [reply resources/language/resource.language.hu_hu/strings.po:12]: Ow. Will fix this. I thought I had restored all headers.

Comment (michaelarnauts) [reply resources/language/resource.language.hu_hu/strings.po]: Fixed!

**Pull request merged**
